### PR TITLE
feat: added contributing guide

### PR DESCRIPTION
**Pull request by alvin-reyes**
# Changes

Added contributing guide

Same changes from the Swagger PR. 

https://github.com/application-research/estuary/pull/107/files

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Contributing to Estuary
+
+Estuary welcomes contributions to our [open source projects on Github](https://github.com/application-research/estuary).
```
Comment (jimmylee) [inline CONTRIBUTING.md:3]: Shouldn't this be a link to `https://github.com/application-research`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Please refer to each project's style and contribution guidelines for submitting patches and additions. In general, we follow the "fork-and-pull" Git workflow.
+
+ 1. **Fork** the repo on GitHub
+ 2. **Clone** the project to your own machine
+ 3. **Commit** changes to your own branch
+ 4. **Push** your work back up to your fork
+ 5. Submit a **Pull request** so that we can review your changes
+
+NOTE: Be sure to merge the latest from "upstream" before making a pull request!
```
Comment (jimmylee) [inline CONTRIBUTING.md:13]: New line

**Review by jimmylee — APPROVED** [review]
LGTM, thanks @alvin-reyes

**Review by gmelodie — APPROVED** [review]
LGTM!

**Pull request merged**